ca/certificate-authority.go

```diff
@@ -62,13 +62,15 @@ type CertificateAuthorityImpl struct {
 // using CFSSL's authenticated signature scheme.  A CA created in this way
 // issues for a single profile on the remote signer, which is indicated
 // by name in this constructor.
-func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config Config) (ca *CertificateAuthorityImpl, err error) {
+func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config Config) (*CertificateAuthorityImpl, error) {
+	var ca *CertificateAuthorityImpl
```

Comment:
Also, it seems like this is the only spot where this comment is relevant, so I'm not going to worry about it.

Thanks for this change, nice improvement!